src/views/Room.vue

```diff
@@ -22,7 +22,7 @@
                   <v-icon>fas fa-fast-forward</v-icon>
                 </v-btn>
                 <vue-slider v-model="volume" style="width: 150px; margin-left: 10px"/>
-                <div style="margin-left: 20px">
+                <div style="margin-left: 20px" class="timestamp">
```

Comment:
Generally, if you have a class for an element you don't want to also have inline styles.

Reply:
I think that instances like this, where I'm setting something like `margin-left` its easier to debug later, and the class can be used in other places.